tests/e2e/specs/api/BuildContainerAPI.spec.ts

```diff
@@ -0,0 +1,162 @@
+import { BrowserTabsUtil } from '../../utils/BrowserTabsUtil';
+import { KubernetesCommandLineToolsExecutor } from '../../utils/KubernetesCommandLineToolsExecutor';
+import { ShellString } from 'shelljs';
+import { ITestWorkspaceUtil } from '../../utils/workspace/ITestWorkspaceUtil';
+import { Dashboard } from '../../pageobjects/dashboard/Dashboard';
+import { ShellExecutor } from '../../utils/ShellExecutor';
+
+suite(`Test podman build container functionality ${BASE_TEST_CONSTANTS.TEST_ENVIRONMENT}`, function (): void {
+	const projectAndFileTests: ProjectAndFileTests = e2eContainer.get(CLASSES.ProjectAndFileTests);
+	const workspaceHandlingTests: WorkspaceHandlingTests = e2eContainer.get(CLASSES.WorkspaceHandlingTests);
+	const loginTests: LoginTests = e2eContainer.get(CLASSES.LoginTests);
+	const browserTabsUtil: BrowserTabsUtil = e2eContainer.get(CLASSES.BrowserTabsUtil);
+	const testWorkspaceUtil: ITestWorkspaceUtil = e2eContainer.get(TYPES.WorkspaceUtil);
+	const dashboard: Dashboard = e2eContainer.get(CLASSES.Dashboard);
+	const shellExecutor: ShellExecutor = e2eContainer.get(CLASSES.ShellExecutor);
+
+	let kubernetesCommandLineToolsExecutor: KubernetesCommandLineToolsExecutor;
+	let workspaceName: string = '';
+	let originalBuildCapabilitiesSetting: string = '';
+	let devSpacesNamespace: string = '';
+	let cheClusterName: string = '';
+
+	const buildPushScript: string = `
+export ARCH=$(uname -m)
+export DATE=$(date +"%m%d%y")
+export USER=$(oc whoami)
+export TKN=$(oc whoami -t)
+export REG="image-registry.openshift-image-registry.svc:5000"
+export PROJECT=$(oc project -q)
+export IMG="\${REG}/\${PROJECT}/hello:\${DATE}"
+
+# Create test directory and Dockerfile
+mkdir -p /projects/dockerfile-test
+cd /projects/dockerfile-test
+
+cat > Dockerfile << EOF
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+RUN echo "Hello from Kubedock!" > /hello.txt
+CMD ["cat", "/hello.txt"]
+EOF
+
+podman login --tls-verify=false --username "\${USER}" --password "\${TKN}" "\${REG}"
+podman build -t "\${IMG}" .
+podman push --tls-verify=false "\${IMG}"
+`;
+
+	const runTestScript: string = `
+export DATE=$(date +"%m%d%y")
+export REG="image-registry.openshift-image-registry.svc:5000"
+export PROJECT=$(oc project -q)
+export IMG="\${REG}/\${PROJECT}/hello:\${DATE}"
+
+oc delete pod test-hello-pod --ignore-not-found
+oc run test-hello-pod --restart=Never --image="\${IMG}"
+
+for i in {1..10}; do
+  PHASE=$(oc get pod test-hello-pod -o jsonpath='{.status.phase}')
+  if [[ "$PHASE" == "Succeeded" ]]; then
+    break
+  elif [[ "$PHASE" == "Failed" ]]; then
+    oc describe pod test-hello-pod
+    exit 1
+  fi
+  sleep 6
+done
+
+oc logs test-hello-pod
+`;
+
+	suiteSetup('Setup DevSpaces with container build capabilities enabled', function (): void {
+		kubernetesCommandLineToolsExecutor = e2eContainer.get(CLASSES.KubernetesCommandLineToolsExecutor);
+		kubernetesCommandLineToolsExecutor.loginToOcp('admin');
```

Comment:
`admin` argument is redundant in here.

kubernetesCommandLineToolsExecutor.loginToOcp() is using `admin` user name by default.

Reply:
thanks for the fixup